modules/light-clients/08-wasm/types/export_test.go

```diff
@@ -22,3 +24,18 @@ func GetClientID(clientStore storetypes.KVStore) (string, error) {
 func NewUpdateProposalWrappedStore(subjectStore, substituteStore storetypes.KVStore, subjectPrefix, substitutePrefix []byte) updateProposalWrappedStore {
 	return newUpdateProposalWrappedStore(subjectStore, substituteStore, subjectPrefix, substitutePrefix)
 }
+
+// WasmQuery wraps wasmQuery and is used solely for testing.
+func WasmQuery[T ContractResult](ctx sdk.Context, clientStore storetypes.KVStore, cs *ClientState, payload QueryMsg) (T, error) {
+	return wasmQuery[T](ctx, clientStore, cs, payload)
+}
+
+// WasmCall wraps wasmCall and is used solely for testing.
+func WasmCall[T ContractResult](ctx sdk.Context, clientStore storetypes.KVStore, cs *ClientState, payload SudoMsg) (T, error) {
+	return wasmCall[T](ctx, clientStore, cs, payload)
+}
+
+// WasmInit wraps wasmInit and is used solely for testing.
+func WasmInit(ctx sdk.Context, clientStore storetypes.KVStore, cs *ClientState, payload InstantiateMessage) error {
```

Comment:
unrelated to this PR but maybe would be nice to align naming with the vm APIs.

- `wasmInstantiate()`
- `wasmQuery()`
- `wasmSudo()`

also happy to keep as is, can change it at a later point since they're all unexported

Reply:
yea. can see benefit in that, I think these have gone through 2 renaming iterations at this point :sweat_smile: